tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
```

Comment:
My total ignorance here: what is tox used for?

Reply:
Multi-version testing!